index.js

```diff
@@ -0,0 +1,67 @@
+console.log("hello, vanilla.");
+
+
+//재시작
+function restart(){
+    input.value="";
+    start();
+    alert("재시작합니다.")
+}
+
+//게임버튼 클릭시 랜덤 숫자 생성
+let start = function(){
+    let numberAll =[0,1,2,3,4,5,6,7,8,9];
+    let ansNum=[];
+    for(let i=0; i<3; i++){
+        let num = numberAll[Math.floor(Math.random()*numberAll.length)];
+        ansNum.push(num);
+    }
+    return ansNum;
+};
+
+let doNum = start();
+console.log(doNum);
+
+//입력값이 세자리 숫자가 아닌 경우 경고창 띄워주기
+let form = document.querySelector("form"),
+    input = document.querySelector("input");
+let intNum =function (){
+    let currentValue = input.value,
+        resultNum =currentValue.split("");
```

Comment:
변수를 선언하실 때 키워드를 붙이는게 가독성에 좋습니다 :D
```suggestion
        const resultNum = currentValue.split("");
```